README.md

```diff
@@ -18,18 +114,65 @@ $ MLNX_OFED_VERSION=50218 /bin/bash build.sh
 ```
 Now the ovs-docker image created successfully
 
+## Pull docker image from docker hub 
+
+  - Install and start docker:
+    ```
+    yum install docker
```

Comment:
Did you verify this?
I think it should be something like this: 
yum install -y yum-utils device-mapper-persistent-data lvm2 mstflint bind-utils
yum-config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo
yum install -y docker-ce docker-ce-cli containerd.io
systemctl enable docker
systemctl start docker
or at least redirect it to docker.com page https://docs.docker.com/engine/install/centos/

Reply:
i only use yum install docker
but maybe its better to redirect to the how to

Reply:
done